src/aap_eda/core/models/rulebook_process.py

```diff
@@ -196,3 +196,29 @@ class Meta:
     line_number = models.IntegerField()
     log = models.TextField()
     log_timestamp = models.BigIntegerField(null=False, default=0)
+
+
+class RulebookProcessQueue(models.Model):
+    """Rulebook Process Queue model.
+
+    Rulebook Process Queue is an instance of ansible-rulebook process
+    that is created when a rulebook process is started.
+    It keeps track of the queue name where the process is running.
```

Comment:
I find this description unclear.  Perhaps something along the line of... An association between a Queue and a RulebookProcess indicating the Queue on which all instances of RulebookProcess will execute.

Reply:
Better.  A couple of tweaks...

    Rulebook Process Queue keeps track of the queue name for a
    Rulebook Process. Every rulebook process is associated with the
    queue, by name, where it ran at creation.

Reply:
updated